packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1049,34 +977,30 @@ async fn test_large_return_data() {
 async fn test_provider_launch_and_connect() {
```

Comment:
I think we ought to keep an E2E test for the `load_sway` and `deploy` workflow. Even if it's not the one that we are going to mainly use, we still need to have it tested somewhere. So keep this test and add another one with the "new" workflow (can be in the same test probably)

Reply:
Since we are keeping the `load_sway` and `deploy` function, we should keep them tested. The question of removing those two functions can indeed be asked, but I think offering more granularity to the users is always nice, especially if the code is already there.

Reply:
Following the discussion on slack, I will create a separate issue to confirm if `load` has other use-cases. But I removed `deploy` to enforce a more streamlined approach via `load_and_deploy`.